### ISSUE TITLE
A question about Read QA

### ISSUE THREAD

**Issue by wolfgangrumpf** [bxlab/metaWRAP#139]
Gherman, I was wondering - if a viral sequence were present in a sample, would it end up in the cleaned reads (to be sorted out later)?  The READ QA folder only contains human sequences pulled out of the raw reads, correct?

**Comment by ursky** [comment]
Yes, virus sequences would still be present in the cleaned reads, and thus the assembly.

**Comment by wolfgangrumpf** [comment]
So in the INITIAL_BINNING, anything that is classified as unbinned would *also* then contain viral sequences?  Other than things like prophage, that is?

**Comment by ursky** [comment]
In theory, yes. Binning is obviously not a reliable way to pull out viral contigs. I strongly recommend following this tutorial: https://www.nature.com/articles/nprot.2017.063

**Comment by wolfgangrumpf** [comment]
I was thinking I could use the unbinned to scan for fungal/viral sequences, since they're (a) not human, and (b) ostensibly not bacterial - or at least strongly filtered out.  I'll take a look at the tutorial - thanks!

**Comment by ursky** [comment]
That would certainly narrow the search and possibly reduce false positives.
